examples/bind.rs

```diff
@@ -0,0 +1,25 @@
+/// Demonstates the use of Hyprland-rs for creating key bindings
+/// and using submaps
+/// 
+/// Usage: cargo run --example bind
+
+use std::io::Read;
+use hyprland::dispatch;
+use hyprland::dispatch::{Dispatch, DispatchType};
+use hyprland::keyword::Keyword;
+//use hyprland::config::binds::Flag;
```

Comment:
remove all the comment here and in the async one too

Reply:
Removed.